src/ValuedTextInput/valuedTextInput.css

```diff
@@ -47,13 +53,14 @@
 
             display:        block;
             position:       absolute;
+            top:            1.1rem;
 
-            top:            0.4rem;
+            width:          0.1rem;
+            height:         1.6rem;
 
-            width:          0.2rem;
-            height:         2.2rem;
+            @extend %Nessie-SemiBold;
```

Comment:
I think maybe this is in the wrong place? (the `::before`/`::after` content is the vertical line, not the label)